cmd/serve.go

```diff
@@ -303,6 +305,33 @@ func sigHandlerConfigReload(config string) {
 	}
 }
 
+func parseIPHostPrefix(host string) (prefixes []netip.Prefix, err error) {
+        //try parsing as prefix
+        prefix, err := netip.ParsePrefix(host)
+        if err == nil {
+                prefixes = append(prefixes, prefix.Masked()) // masked and canonical for easy of debugging, shouldn't matter
+                return prefixes, nil                         // success
+        }
```

Comment:
For example, if someone enters 10.1.2.3/8, the canonical address is 10.0.0.0/8. It represents every address from 10.0.0.1 to 10.255.255.255. If the address is ever logged in the future, this would help the output be more consistent.

Should I explain this more in the comment?